apm-agent-plugins/apm-dubbo-plugin/src/main/java/co/elastic/apm/agent/dubbo/AlibabaMonitorFilterInstrumentation.java

```diff
@@ -44,6 +46,9 @@ public AlibabaMonitorFilterInstrumentation(ElasticApmTracer tracer) {
         return named("com.alibaba.dubbo.monitor.support.MonitorFilter");
     }
 
+    /**
+     * {@link org.apache.dubbo.monitor.support.MonitorFilter#invoke(Invoker, Invocation)}
```

Comment:
Should it be `{@link com.alibaba.dubbo.monitor.support.MonitorFilter#invoke(Invoker, Invocation)}` ?

Reply:
Good catch!